src/main/ts/api/Dts.ts

```diff
@@ -0,0 +1,83 @@
+import { Plugin } from 'rollup';
+import rollupDts, { Options} from 'rollup-plugin-dts';
+import * as ts from 'typescript';
+import { clean, CleanerOptions } from '../ast/ts/DtsCleaner';
+import { parse } from '../ast/ts/Parser';
+import { fail } from '../utils/Fail';
+
+interface DtsOptions extends CleanerOptions, Options {
```

Comment:
Multiple inheritance makes me a little nervous... It doesn't clash now, but will the compiler give us any form of hard error if either of those interfaces start to clash later?

Reply:
We do this all the time, so it's fine (if it was an actual implementation that'd be a different story). Also yes it will complain.